Makefile

```diff
@@ -7,7 +7,7 @@ PROFILE=release
 XAPI_VERSION ?= $(shell git describe --always --dirty || echo "NO_GIT")
 MANDIR ?= $(OPTDIR)/man/man1/
 
-.PHONY: build clean test doc python format list-hd install uninstall
+.PHONY: build clean test doc python format install uninstall
```

Comment:
```suggestion
.PHONY: build clean test doc python format quality-gate install uninstall
```

Reply:
The quality-gate target is already declared as .PHONY (see the declaration)